### ISSUE TITLE
Minecraft crash on research

### ISSUE THREAD

**Issue by Rinma** [TheRActivator/From-the-Ground-Up#25]
When i try to research carpentary and i place the parchment in the Research Table the game crash with this error message:

```
---- Minecraft Crash Report ----

WARNING: coremods are present:
  IC2core (industrialcraft-2-2.7.59-ex111.jar)
  AppleCore (AppleCore-mc1.11.2-2.1.2.jar)
  AdvancedRocketryPlugin (AdvancedRocketry-1.11.2-1.2.1.jar)
  CCLCorePlugin (CodeChickenLib-1.11.2-2.6.0.254-universal.jar)
  Extra Alchemy ASM (extraalchemy-0.3.4-release.jar)
  LoadingPlugin (RandomThings-MC1.11.2-4.0.jar)
  Transformer (EnhancedChat-1.11.2-1.1.1.jar)
  BookshelfLoadingPlugin (Bookshelf-1.11.2-2.0.0.385.jar)
  Inventory Tweaks Coremod (InventoryTweaks-1.62.jar)
  RBLoadingPlugin (RealBench-1.11.2-1.3.2.jar)
  TransformerLoader (OpenComputers-MC1.11.2-1.7.0.24.jar)
  TANLoadingPlugin (ToughAsNails-1.11-2.0.5.jar)
Contact their authors BEFORE contacting forge

// I blame Dinnerbone.

Time: 6/15/17 10:54 AM
Description: Updating screen events

java.lang.NullPointerException: Updating screen events
	at ftgumod.gui.researchtable.ContainerResearchTable.hasRecipe(ContainerResearchTable.java:106)
	at ftgumod.gui.researchtable.ContainerResearchTable.func_75130_a(ContainerResearchTable.java:156)
	at ftgumod.gui.researchtable.ContainerResearchTable.func_184996_a(ContainerResearchTable.java:201)
	at net.minecraft.client.multiplayer.PlayerControllerMP.func_187098_a(PlayerControllerMP.java:551)
	at net.minecraft.client.gui.inventory.GuiContainer.func_184098_a(GuiContainer.java:632)
	at net.minecraft.client.gui.inventory.GuiContainer.func_146286_b(GuiContainer.java:598)
	at net.minecraft.client.gui.GuiScreen.func_146274_d(GuiScreen.java:554)
	at net.minecraft.client.gui.GuiScreen.func_146269_k(GuiScreen.java:512)
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1695)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1056)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:372)
	at net.minecraft.client.main.Main.main(SourceFile:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Client thread
Stacktrace:
	at ftgumod.gui.researchtable.ContainerResearchTable.hasRecipe(ContainerResearchTable.java:106)
	at ftgumod.gui.researchtable.ContainerResearchTable.func_75130_a(ContainerResearchTable.java:156)
	at ftgumod.gui.researchtable.ContainerResearchTable.func_184996_a(ContainerResearchTable.java:201)
	at net.minecraft.client.multiplayer.PlayerControllerMP.func_187098_a(PlayerControllerMP.java:551)
	at net.minecraft.client.gui.inventory.GuiContainer.func_184098_a(GuiContainer.java:632)
	at net.minecraft.client.gui.inventory.GuiContainer.func_146286_b(GuiContainer.java:598)
	at net.minecraft.client.gui.GuiScreen.func_146274_d(GuiScreen.java:554)
	at net.minecraft.client.gui.GuiScreen.func_146269_k(GuiScreen.java:512)

-- Affected screen --
Details:
	Screen name: ftgumod.gui.researchtable.GuiResearchTable
```

**Comment by Astavie** [comment]
Thanks for the report. I think I already know what the problem is

**Comment by Astavie** [comment]
Should be fixed in 1.1.6
